mysqld_exporter.go

```diff
@@ -134,6 +136,7 @@ const (
 	performanceSchema = "perf_schema"
 	slaveStatus       = "slave_status"
 	binlog            = "binlog"
+	tokudb            = "tokudb"
```

Comment:
Maybe we should call the `engine` subsystem, and prefix the metric name with tokudb.  Then we get `mysql_engine_tokudb`.  This leaves us room to implement metrics from other DB engines.